id/documentation/success-stories/index.md

```diff
@@ -9,85 +9,85 @@ yang menggunakannya sebagai hobi. Di sini Anda akan menemukan sebagian
-
-* [A List Apart][3], majalah digital bagi para webmaster yang telah
-  berdiri sejak tahun 1997, telah direnovasi dan menggunakan aplikasi
-  khusus yang dikembangkan menggunakan Ruby on Rails.
-
-* [Twitter][4], situs komunitas yang sangat ramai menggunakan Ruby on
-  Rails sebagai *framework*.
-
-* [BookJetty][5] merupakan situs bagi pecinta buku, dikembangkan oleh
-  [Herryanto Siatono][6] ([Pluit Solutions][7]).
-
-* [Ruby on Rails Applications][8] berisi sebagian kecil daftar aplikasi
-  yang dikembangkan menggunakan Ruby on Rails.
-
-* [Rails 100][9] berisi daftar 100 aplikasi Rails terpopuler
-  berdasarkan statistik dari Alexa.
-
-* [Real World Usage di Rails Wiki][10] berisi daftar aplikasi berbasis
-  Ruby on Rails.
-
 #### Simulasi
 
-* [NASA Langley Research Center][11] menggunakan Ruby untuk melakukan
+* [NASA Langley Research Center][1] menggunakan Ruby untuk melakukan
   simulasi.
 
-* Kelompok riset di [Motorola][12] menggunakan Ruby untuk melakukan
+* Sebuah kelompok riset di [Motorola][2] menggunakan Ruby untuk melakukan
   simulasi, baik untuk meng-*generate* skenario maupun memproses data
   hasilnya.
 
+#### Pemodelan 3D
+
+* [Google SketchUp][3] adalah sebuah aplikasi pemodelan 3D yang menggunakan
+  Ruby untuk *macro scripting API*.
+
 #### Bisnis
 
-* [Toronto Rehab][13] menggunakan aplikasi berbasis RubyWebDialogs dalam
+* [Toronto Rehab][4] menggunakan aplikasi berbasis RubyWebDialogs dalam
   manajemen dukungan *on-call* dan *on-site* untuk *help desk* Teknologi
   Informasi dan tim operasi Teknologi Informasi.
 
 #### Robotika
 
-* Proyek [MORPHA][14] menggunakan Ruby untuk mengimplementasikan bagian
+* Proyek [MORPHA][5] menggunakan Ruby untuk mengimplementasikan bagian
   pengendali reaktif dari robot Siemens yang digunakan.
 
 #### Jaringan Komputer
 
-* [Open Domain Server][15] menggunakan Ruby untuk memungkinkan pengguna
+* [Open Domain Server][6] menggunakan Ruby untuk memungkinkan pengguna
   Dynamic DNS meng-*update* konfigurasi IP mereka secara *real time*
   agar dapat dipetakan ke domain statis.
 
 #### Telekomunikasi
 
-* Ruby digunakan oleh [Lucent][16] pada produk wireless 3G yang mereka
+* Ruby digunakan oleh [Lucent][7] pada produk wireless 3G yang mereka
   kembangkan.
 
 #### Administrasi Sistem
 
 * Ruby digunakan untuk mengembangkan koleksi data sentral di [Level 3
-  Communications][17] pada sistem *Unix Capacity and Planning* yang
+  Communications][8] pada sistem *Unix Capacity and Planning* yang
   memproses statistik di lebih dari 1700 server Unix (Solaris dan Linux)
   yang tersebar di seluruh dunia.
 
+#### Aplikasi Web
+
+* [Basecamp][9], sebuah aplikasi manajemen proyek berbasis *web* yang
+  dikembangkan oleh [37signals][10], diprogram sepenuhnya dalam bahasa Ruby.
 
+* [A List Apart][11], sebuah majalah digital bagi para *webmaster* yang telah
+  berdiri sejak tahun 1997, telah direnovasi dan menggunakan aplikasi
+  khusus yang dikembangkan menggunakan Ruby on Rails.
 
```

Comment:
maybe we can put Github and Shopify here?

Reply:
Hmm 🤔

The English version of success stories doesn't have that. Shouldn't we follow that version?

Reply:
If would like to do so, please change upstream (English) first.

Reply:
I will do the changes on upstream later, I think we can merge this PR first?